.jshintrc

```diff
@@ -11,11 +11,7 @@
   "newcap": true,
   "noarg": true,
   "quotmark": "single",
-  "regexp": true,
   "undef": true,
   "unused": true,
-  "strict": true,
-  "trailing": true,
-  "smarttabs": true,
-  "white": true
+  "strict": true
```

Comment:
Move all the generator code to `src/` and then compile using babel into `generator/` on `npm publish`. Or something. I'm ok with holding off on doing this.